deploy/docs/v2_migration_doc.md

```diff
@@ -82,3 +82,65 @@ the exact steps for migration.
 ## How to upgrade
 
 **Note: The below steps are using Helm 3. Helm 2 is not supported.**
+
+### 1. Upgrade to helm chart version `v1.3.2`
+
+Run the below command to fetch the latest helm chart:
+
+```bash
+helm repo update
```

Comment:
I think that before this step we can add:
Please make sure that you have sumologic helm repo configured, check this using `helm repo list`. If sumologic helm repo is not configured use `helm repo add sumologic https://sumologic.github.io/sumologic-kubernetes-collection`

Maybe it'll make someone's life easier ;) When I deployed collection 1.3.2 from sources I don't have any helm repo.

Reply:
Added